0.generate-profiles/pipelines/readme.md

```diff
@@ -0,0 +1,11 @@
+To generate profiles, two CellProfiler pipelines were run for each batch.
+
+Images are first loaded into the `illumination_correction` pipeline, which is used to generate per-plate illumination correction functions.
+This pipeline will export .npy files for use in the subsequent analysis pipeline.
+
+Following this, the images and the illumination correction .npy files are loaded into the `analysis` pipeline. This pipeline will perform
+the measurements and export them into .csv output files.
```

Comment:
```suggestion
Following this, the images and the illumination correction .npy files are loaded into the `analysis` pipeline. This pipeline will perform the measurements and export them into .csv output files.
```

Minor adjustment to use the [one-sentence-per-line paradigm](https://news.ycombinator.com/item?id=4642395).